### ISSUE TITLE
Formatting of interface generic method is unclear.

### ISSUE THREAD

**Issue by lukaswelinder** [tgreyjs/typedoc-plugin-markdown#38]
I'd be happy to sort this out myself and make a PR, but I'm a bit lost as to what part of the codebase is causing this issue.

**The following is related to github MD, I have not tried reproducing this in other styles.**

When parsing an interface's method that is generic I get the following:

```
**reduce**R(reducer: *`function`*, initialReduction?: *`R`*, context?: *`any`*): `R`
```

The expected output should be something like this:

```
**reduce**<`R`>(reducer: *`function`*, initialReduction?: *`R`*, context?: *`any`*): `R`
```

I'd appreciate any input, or at least a nudge in the right direction, thanks.

**Comment by tgreyjs** [comment]
Thanks - have released v1.1.15 that should fix this. FYI if you want to make any contributions theme template code is located in src/theme. Take a look at http://typedoc.org/guides/themes/#custom-themes for a brief explanation.

**Comment by lukaswelinder** [comment]
@tgreyjs Got it, thanks!
